it/test/uk/gov/hmrc/tradergoodsprofiles/controllers/CreateRecordControllerIntegrationSpec.scala

```diff
@@ -118,6 +118,52 @@ class CreateRecordControllerIntegrationSpec
       }
     }
 
+    "return BadRequest when Accept header is invalid" in {
+      withAuthorizedTrader()
+
+      val headers = Seq("X-Client-ID" -> "clientId", "Content-Type" -> "application/json")
+      val result  = await(
+        wsClient
+          .url(url)
+          .withHttpHeaders(headers: _*)
+          .post(requestBody)
+      )
+
+      result.status mustBe BAD_REQUEST
+      result.json mustBe createExpectedError(
+        "INVALID_HEADER_PARAMETER",
+        "Accept was missing from Header or is in wrong format",
+        4
+      )
+    }
+
+    "return BadRequest when Content-Type header is empty" in {
+      withAuthorizedTrader()
+
+      val headers = Seq("X-Client-ID" -> "clientId", "Content-Type" -> "", "Accept" -> "application/vnd.hmrc.1.0+json")
+      val result  = await(
+        wsClient
+          .url(url)
+          .withHttpHeaders(headers: _*)
+          .post(requestBody)
+      )
+
+      result.status mustBe UNSUPPORTED_MEDIA_TYPE
```

Comment:
can we assert bad request content type error response here please?

Reply:
Or rather change the test description to "return Unsupported media type"

Reply:
I have addressed the above